tests/fixtures/agent.conf

```diff
@@ -1,7 +1,7 @@
 agent {
     data_dir = "$AGENT_DATA_DIR"
-    log_level = "ERROR"
-    log_file = "/dev/null"
+    log_level = "DEBUG"
+    # log_file = "/dev/null"
```

Comment:
Just wanted to check, was this supposed to be commented out or left over from debugging?

Reply:
Good catch, I thought I put these back to what they were, definitely left over from debugging, thanks!